### ISSUE TITLE
Localized FileEdit

### ISSUE THREAD

**Issue by 1gnacio** [Megabit/Blazorise#4782]
**Describe the bug**
When using FileEdit component the 'Select file' and 'No elements selected' labels appear in the browser UI language instead of the website language

**To Reproduce**
1. Render <FileEdit /> component
2. Input is rendered in the browser UI language instead of the website defined language

**Expected behavior**
Labels should be rendered in the website language / culture, instead of the UI language

**Screenshots**
As shown in the [Localization docs](https://blazorise.com/docs/helpers/localization) the 'Select file' changes but the placeholder does not

![image](https://github.com/Megabit/Blazorise/assets/38591482/a5ec7cae-dba9-4ca3-b45d-9d99ac597f2e)

**Additional context**
I'm currently using Blazorise from NuGet package version 1.0.4
In this version the input label also does not change with the website current culture

![image](https://github.com/Megabit/Blazorise/assets/38591482/5326b967-938c-472d-9a18-6bfd9aaf323e)

**Comment by David-Moreira** [comment]
Unfortunately this is a known "issue". our base components always try to use the equivalent html native components and as such the native `input type="file"` makes use of the localization of the browser which cannot be programmatically changed.

We can take this as a feature and think about doing a customized version or just mimick the original input file look but with proper localization, @stsrki ?

**Comment by 1gnacio** [comment]
Thanks for your help, maybe it isn't related to this bug but another option is to implement a component that can show a preview of the file uploaded with some *load* and *delete* buttons (icon buttons?) so the original input can be hidden.

**Comment by David-Moreira** [comment]
Take a look at https://blazorise.com/docs/components/file-picker which has more inbuilt features like the one you mention. (It still builds upon `FileEdit` and therefore input type=file so you'll still have the same localization problem though.)
![image](https://github.com/Megabit/Blazorise/assets/22283161/d39cadab-b368-44f1-aa5e-cd779b48fb53)

**Comment by 1gnacio** [comment]
I've seen already your FilePicker component and it does not solve my problem with the localization. I had these two in mind (more related to the feature I'm working on)

This one is from [Metronic template](https://preview.keenthemes.com/metronic8/demo1/account/settings.html)

![image](https://github.com/Megabit/Blazorise/assets/38591482/4d776e0a-bdfb-46e2-8a6d-318baf57d06e)

Or maybe this one from Azure DevOps

![image](https://github.com/Megabit/Blazorise/assets/38591482/ecc22056-c67e-42f1-909d-46aa20828a45)

Both implementations (as I've seen in the HTML skeleton) hide the input tag to render their own implementation, and both look fine.

**Comment by David-Moreira** [comment]
Yes, we acknowledge that currently both components will not help you with the localization. 
But we can definitely improve on that. 

We've already taken the approach of providing both a native html component and a more customized component in the past, i.e `NumericEdit`(uses native input) vs `NumericPicker` (uses a customized version based on a popular js lib). 
So maybe we can take the same approach here.

@stsrki What are your thoughts? I was thinking that maybe we could refactor the newer `FilePicker` to solve this inherit problem and have the same design decision as the `NumericEdit` vs `NumericPicker` for instance.

**Comment by David-Moreira** [comment]
If you still want to use the core implementations however, maybe you can do something like : 
Set the `FileEdit` Style to `"display:none;"` and then roll your look with localization. Then call the `FileEdit` `ShowPicker` on click.
But I understand if that's cumbersome.

**Comment by 1gnacio** [comment]
I understand that might be a workaround for this scenario however I feel that it isn't the intended use for your components, also it could be a problem in the future if someone else has to maintain it. I'm trying to use your components as clean as possible, so it can be easily updated in the future if it has to.

As I've been researching about this issue it looks like this tag won't accept custom labels (in the near future at least) so I assume hiding it is the way to do it, instead of relying on the browser language.
